ee/tables/tablewrapper/tablewrapper.go

```diff
@@ -145,6 +145,7 @@ func (wt *wrappedTable) generate(ctx context.Context, queryContext table.QueryCo
 			"query_start_time", queryStartTime.String(),
 			"query_timeout", genTimeout.String(),
 		)
+		observability.TablewrapperTimeoutCounter.Add(ctx, 1)
```

Comment:
Should we flag the trace as having an error?

Reply:
You got it! https://github.com/kolide/launcher/pull/2238